README.md

```diff
@@ -1,5 +1,81 @@
-# tailwindcss-vertical-rhythm
+# Tailwindcss-vertical-rhythm
 
 A tailwind plugin which helps create scaling typography.
 
-TODO setup issues, readme, and maybe a better contributing md.
+# Installation
+You can install the library using your prefered package manager
+
+## NPM
+```sh
+npm install tailwindcss-vertical-rhythm
+```
+## Yarn
+```sh
+yarn add tailwindcss-vertical-rhythm
+```
+## PNPM
+```sh
+pnpm add tailwindcss-vertical-rhythm
+```
+# Configuration
+
+Add `tailwindcss-vertical-rhythm` as a plugin in `tailwind.config.js`
+
+```js
+module.exports = {
+  //further configurations
+  plugins: [require("tailwindcss-vertical-rhythm")]
+}
+```
+
+## Extracting Classes
+
+You can use a tailwind.css file for extracting classes and reuse code
+
+```css
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer components {
+  //name of class
+  .blue {
+    //css that will be applied
+    @apply py-2 px-4 bg-blue-500 text-white font-semibold rounded-lg shadow-md hover:bg-blue-700 focus:outline-none focus:ring-2 focus:ring-blue-400 focus:ring-opacity-75;
+  }
+}
+```
+Learn more about extracting classes [here](https://tailwindcss.com/docs/extracting-components)
+
+## Scale options
```

Comment:
The scale feature of tailwind isn't relevant here so I'd prefer to see some plugin config overview.

I need to go through and document the API (tonight's job), For the moment you can use something like this:

This plugin set's the root font-size of the site to 16px and prefers `rem` units.

You can customize this behavior with:

```
  verticalRhythm: {
    basePixel: 14, // Change the base unit to 14px. Pass a number not a px value
    unit: "em"
  },
```

Reply:
Changed this to the relevant feature